addons/payment_adyen/models/payment_transaction.py

```diff
@@ -194,3 +195,53 @@ def _adyen_tokenize_from_feedback_data(self, data):
         _logger.info(
             "created token with id %s for partner with id %s", token.id, self.partner_id.id
         )
+
+    def _send_refund_request(self, **kwargs):
+        """ Override of payment to send a refund request to Adyen.
+        Note: self.ensure_one()
+        :return: None
+        """
+        super()._send_refund_request(**kwargs)
+        if self.provider != 'adyen':
+            return
+
+        refund_amount = kwargs.get('refund_amount', 0)
```

Comment:
Even if you don't replace `**kwargs` by `refund_amount=None` as suggested above, this isn't necessary. You can totally extract a given param from the kwargs. See the signature of the overrides of `_get_compatible_acquirers` for example. You just need to pass the param back to `super` because it has been extracted from the kwargs.